netzob/src/netzob/Export/WiresharkDissector/WiresharkDissector.py

```diff
@@ -0,0 +1,560 @@
+# |                                                                           |
+# | This program is distributed in the hope that it will be useful,           |
+# | but WITHOUT ANY WARRANTY; without even the implied warranty of            |
+# | MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the              |
+# | GNU General Public License for more details.                              |
+# |                                                                           |
+# | You should have received a copy of the GNU General Public License         |
+# | along with this program. If not, see <http://www.gnu.org/licenses/>.      |
+# +---------------------------------------------------------------------------+
+# | @url      : http://www.netzob.org                                         |
+# | @contact  : contact@netzob.org                                            |
+# | @sponsors : Amossys, http://www.amossys.fr                                |
+# |             Supélec, http://www.rennes.supelec.fr/ren/rd/cidre/           |
+# +---------------------------------------------------------------------------+
+
+# +---------------------------------------------------------------------------+
+# | Standard library imports
+# +---------------------------------------------------------------------------+
+import errno
+from gettext import gettext as _
+from operator import methodcaller
+import re
+import binascii
+import random
+# +---------------------------------------------------------------------------+
+# | Related third party imports
+# +---------------------------------------------------------------------------+
+
+
+# +---------------------------------------------------------------------------+
+# | Local application imports
+# +---------------------------------------------------------------------------+
+from netzob.Export.WiresharkDissector.CodeBuffer import LUACodeBuffer
+from netzob.Model.Vocabulary.Messages.L2NetworkMessage import L2NetworkMessage
+from netzob.Model.Vocabulary.Messages.L3NetworkMessage import L3NetworkMessage
+from netzob.Model.Vocabulary.Messages.L4NetworkMessage import L4NetworkMessage
+from netzob.Export.WiresharkDissector.WiresharkFilter import WiresharkFilterFactory
+from netzob.Model.Vocabulary.AbstractField import AbstractField
+from netzob.Model.Vocabulary.Types.AbstractType import AbstractType
+from netzob.Model.Vocabulary.Types.Raw import Raw
+from netzob.Model.Vocabulary.Types.ASCII import ASCII
+from netzob.Model.Vocabulary.Types.HexaString import HexaString
+from netzob.Model.Vocabulary.Types.TypeConverter import TypeConverter
+from netzob.Common.Utils.Decorators import typeCheck, NetzobLogger
+
+
+
+@NetzobLogger
+class WiresharkDissector(object):
+    """
+        Usage: >>> WiresharkDissector.dissectSymbols(symbols,'outputFilenane')
+        or for single symbols
+        Usage: >>> WiresharkDissector.dissectSymbol(symbol,'outputFilenane')
```

Comment:
Could you add unit tests as doctests, in order to validate the provided code.

Besides, thanks for having getting back the old code from 0.4.2 ;)